services/ml/compilation_delegate_bnns.mm

```diff
@@ -525,6 +527,49 @@ void SetExtendInputs(const uint32_t extend_input_idx,
   return true;
 }
 
+bool CompilationDelegateBnns::CompileLogistic(
+    const mojom::ModelInfoPtr& model,
+    const mojom::OperationPtr& operation,
+    OperationMac& operation_bnns) {
+  DLOG(INFO) << "CompilationDelegateBnns::CompilLogistic";
+
+  operation_bnns.offset_x = 0;
+  operation_bnns.offset_y = 0;
+
+  BNNSVectorDescriptor in_desc, out_desc;
+  const std::vector<uint32_t>& inputs = operation->inputs;
+  const std::vector<uint32_t>& outputs = operation->outputs;
+  const mojom::OperandPtr& input = model->operands[inputs[0]];
+  const mojom::OperandPtr& output = model->operands[outputs[0]];
+
+  operation_bnns.input_batch_size = input->dimensions[0];
+
+  int32_t size = 1;
+  for (size_t i = 1; i < input->dimensions.size(); i++) {
+    size = size * input->dimensions[i];
+  }
+  in_desc.size = size;
+  in_desc.data_type = BNNSDataTypeFloat32;
+  in_desc.data_scale = 0;
+  in_desc.data_bias = 0;
+  size = 1;
+  for (size_t i = 1; i < output->dimensions.size(); i++) {
+    size = size * output->dimensions[i];
+  }
+  out_desc.size = size;
+  out_desc.data_type = BNNSDataTypeFloat32;
+  out_desc.data_scale = 0;
+  out_desc.data_bias = 0;
+  BNNSActivation activation;
+  bzero(&activation, sizeof(activation));
```

Comment:
Maybe it's easy to use [= {}](https://en.cppreference.com/w/cpp/language/value_initialization) to zero initialization

Reply:
Ok.